src/main/webapp/resources/js/pages/sequence-files/components/FastQCCharts.jsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file renders the FastQC charts component.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Col, Layout , Row, Typography} from "antd";
+import { SPACE_MD } from "../../../styles/spacing";
+import { grey1 } from "../../../styles/colors";
+import { TabPaneContent } from "../../../components/tabs/TabPaneContent";
+import { getFastQCImages } from "../../../apis/files/sequence-files";
+import { ContentLoading } from "../../../components/loader";
+import {
+  seqObjId,
+  seqFileId
+} from "../fastqc-constants";
+
+import styled from "styled-components";
+
+const StyledImage = styled.img`
+  height: 100%;
+  width: 100%;
```

Comment:
Can you add:
```
  border: 1px solid ${grey4};
  padding: ${SPACE_MD};
  border-radius: 2px;
```

Reply:
Added in 139845ae0399cc73ea9aa42936a8341d24590480